can_tools/scrapers/official/TX/texas_vaccine.py

```diff
@@ -0,0 +1,113 @@
+import pandas as pd
+import us
+import lxml.html
+import requests
+from can_tools.scrapers import CMU
+from can_tools.scrapers.official.base import StateDashboard
+
+# the crename keys became absurdly long so I store them in another file
```

Comment:
You could potentially do things via a loop, but dumping them into another file like this to keep it simple seems fine with me.

Reply:
Resolved because TX changed the data format to long.